### DISCUSSION TITLE
Nginx configuration

### DISCUSSION THREAD

**Discussion by WeWatchYourWebsite** [YOURLS/YOURLS#3050] [Q&A] [answered]
I have followed the troubleshooting steps.

I have YOURLS in a folder /enc, I have added a new Nginx conf file with this:

location /enc {
        try_files $uri $uri/ /yourls-loader.php$is_args$args;
}

And I keep getting 404's. The database is working as I see the data there, but it won't redirect.

**Comment by WeWatchYourWebsite** [comment, marked as answer]
I found my own answer.

Instead of this:

location /enc {
try_files $uri $uri/ /yourls-loader.php$is_args$args;
}

I added the folder like this:

location /enc {
try_files $uri $uri/ /enc/yourls-loader.php$is_args$args;
}

Then did nginx -t
and then reloaded and it works. 

I hope this helps someone else.

**Reply by ozh** [reply]
I've updated https://github.com/YOURLS/YOURLS/wiki/Nginx-configuration in case this helps other folks.
